portfolio/src/main/webapp/script.js

```diff
@@ -11,18 +11,17 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
+ * This is just a modification from random greeting
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+function addRandomFact(){
+  const facts=['I joined Google during the Covid crisis', 
```

Comment:
nit: should be formatted a little differently - I think if you put a comma after the last entry and run a linter, it should format automatically